File: ts/wam/3932.ts
export const WamWebcButterbarEvent = 3932 as const

export interface WebcButterbarEventEvent {
  /** field 1, wire `webc_butterbar_action` */
  webcButterbarAction?: WEBCBUTTERBARACTIONTYPE
  /** field 2, wire `webc_butterbar_type` */
  webcButterbarType?: WEBCBUTTERBARBBTYPE
}

File: rust/wam/3932.rs
pub const WAM_WEBC_BUTTERBAR_EVENT: u32 = 3932;

#[derive(Debug, Default)]
pub struct WebcButterbarEventEvent {
    /// field 1, wire `webc_butterbar_action`
    pub webc_butterbar_action: Option<WEBCBUTTERBARACTIONTYPE>,
    /// field 2, wire `webc_butterbar_type`
    pub webc_butterbar_type: Option<WEBCBUTTERBARBBTYPE>,
}